packages/client/hmi-client/src/components/widgets/tera-input-field.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+	<div
+		class="input-container"
+		:class="{ error: errorMessage, disabled: disabled, focused: focused }"
+		@click="focusInput"
```

Comment:
Making sure this click event is not coming from elsewhere: 
- https://vuejs.org/guide/essentials/event-handling.html#event-modifiers
```suggestion
		@click.self.stop="focusInput"
```

Reply:
I've added this to the label as well to focus when that it clicked